equipment-datastore-common-test/src/main/java/com/telecominfraproject/wlan/equipment/datastore/BaseEquipmentDatastoreTest.java

```diff
@@ -348,6 +348,141 @@ public void testEquipmentPagination()
+       
+       // Check search on name
+       PaginationContext<Equipment> context_name = new PaginationContext<>(10);
+       PaginationResponse<Equipment> page1_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, context_name);
+       PaginationResponse<Equipment> page2_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page1_name.getContext());
+       PaginationResponse<Equipment> page3_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page2_name.getContext());
+       PaginationResponse<Equipment> page4_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page3_name.getContext());
+       PaginationResponse<Equipment> page5_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page4_name.getContext());
+       PaginationResponse<Equipment> page6_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page5_name.getContext());
+       PaginationResponse<Equipment> page7_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page6_name.getContext());
+       
+       //verify search on name and mac returned the same items
+       assertEquals(page1.getItems(), page1_name.getItems());
+       assertEquals(page2.getItems(), page2_name.getItems());
+       assertEquals(page3.getItems(), page3_name.getItems());
+       assertEquals(page4.getItems(), page4_name.getItems());
+       assertEquals(page5.getItems(), page5_name.getItems());
+       assertEquals(page6.getItems(), page6_name.getItems());
+       assertEquals(page7.getItems(), page7_name.getItems());
+       
+       List<String> expectedPage3Strings = new ArrayList<>(Arrays.asList(new String[]{"qr_27", "qr_28", "qr_29", "qr_3", "qr_30", "qr_31", "qr_32", "qr_33", "qr_34", "qr_35" }));
+       List<String> actualPage3Strings = new ArrayList<>();
+       page3.getItems().stream().forEach( ce -> actualPage3Strings.add(ce.getName()) );
+       
+       assertEquals(expectedPage3Strings, actualPage3Strings);
+       
+       //test first page of the results with empty sort order -> default sort order (by Id ascending)
+       PaginationResponse<Equipment> page1EmptySort = testInterface.getForCustomer(customerId_1, Collections.emptyList(), context);
+       assertEquals(10, page1EmptySort.getItems().size());
+
+       List<String> expectedPage1EmptySortStrings = new ArrayList<>(Arrays.asList(new String[]{"qr_0", "qr_1", "qr_2", "qr_3", "qr_4", "qr_5", "qr_6", "qr_7", "qr_8", "qr_9" }));
+       List<String> actualPage1EmptySortStrings = new ArrayList<>();
+       page1EmptySort.getItems().stream().forEach( ce -> actualPage1EmptySortStrings.add(ce.getName()) );
+
+       assertEquals(expectedPage1EmptySortStrings, actualPage1EmptySortStrings);
+
+       //test first page of the results with null sort order -> default sort order (by Id ascending)
+       PaginationResponse<Equipment> page1NullSort = testInterface.getForCustomer(customerId_1, null, context);
```

Comment:
should test testInterface.searchByMacAndName, not testInterface.getForCustomer in here